runsc/boot/fs.go

```diff
@@ -230,6 +237,10 @@ func isSupportedMountFlag(fstype, opt string) bool {
 	switch opt {
 	case "rw", "ro", "noatime", "noexec":
 		return true
+	case "bind", "rbind":
+		if fstype == nonefs || !specutils.ContainsStr(supportedFilesystems, fstype) {
+			return true
+		}
```

Comment:
This check is not needed, because these options do not change mount behavior. Please remove.